bench/tiered/push_pull.c

```diff
@@ -214,11 +218,15 @@ recover_validate(const char *home, uint32_t num_records, uint64_t file_size, int
     testutil_wiredtiger_open(opts, home, buf, NULL, &conn, true, true);
     testutil_check(conn->open_session(conn, NULL, NULL, &session));
 
-    // srand((unsigned long)getpid() + num_records);
-    srand((uint32_t)getpid() + num_records);
+    /* Seed the random number generator */
+    v = (uint32_t)getpid() + num_records + (2 * counter);
+    w = (uint32_t)(2 * flush) + num_records;
+    z = (2 * counter) + num_records;
+    __wt_random_init_custom_seed(&rnd, v, w, z);
+
     str_len = sizeof(data_str) / sizeof(data_str[0]);
     for (i = 0; i < str_len - 1; i++)
-        data_str[i] = 'a' + (uint32_t)rand() % 26;
+        data_str[i] = 'a' + (uint32_t)__wt_random(&rnd) % 150;
```

Comment:
We'll probably come up with something different (see the other comment where `... % 150` is used).  So probably good to have a single function to populate an array, called by these two places.

Reply:
Fixed